uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1652,6 +1653,81 @@ sub computeSNR {
     }
 }
 
+################################################################
+#######################  computeDeepQC #############################
+################################################################
+sub computeDeepQC {
+    my $this = shift;
+    my ($row, $filename, $fileID, $base, $fullpath, $message);
+    my ($tarchiveID, $tarchive_srcloc, $profile)= @_;
+    my $data_dir = NeuroDB::DBI::getConfigSetting(
+                        $this->{dbhr},'dataDirBasepath'
+                        );
+    my $upload_id = getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc);
+    my $ua = LWP::UserAgent->new;
+    my $json = 'JSON::PP'->new;
+    # note: url must be changed to production version later
+    # note 2: include DeepQC modality in prod file
+    my $acqIDForT1Scan = 44;
```

Comment:
this `44` should be generic :)

Reply:
yeah I know...the thing is that Andrew's model only works for IBIS data at the moment anyway. But I can update it to be generic.

Reply:
yes, but because it is making it to main Loris-MRi repo, it should be generic.
I believe you cna get this feidl from the ConfigSettings/Imaging Pipeline section. I believe the DTI PreProcessing pipeline needs the T1W modality as an input, so you can get it from there.
Am I not correct @cmadjar?

Reply:
@MounaSafiHarab @llevitis indeed, you could grep it from the config setting called t1_scan_type :)